src/components/PeopleFilters.tsx

```diff
@@ -1,12 +1,47 @@
+import classNames from 'classnames';
+import { useSearchParams } from 'react-router-dom';
+import { SearchLink } from './SearchLink';
+import { getSearchWith } from '../utils/searchHelper';
+
 export const PeopleFilters = () => {
+  const [searchParams, setSearchParams] = useSearchParams();
+  const sex = searchParams.get('sex');
+  const query = searchParams.get('query') || '';
+  const centuries = searchParams.getAll('centuries');
+
+  const centuriesNames = ['16', '17', '18', '19', '20'];
+
+  const handleQuary = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setSearchParams(
+      getSearchWith(searchParams, { query: e.target.value || null }),
+    );
+  };
+
   return (
     <nav className="panel">
       <p className="panel-heading">Filters</p>
 
       <p className="panel-tabs" data-cy="SexFilter">
-        <a className="is-active" href="#/people">All</a>
-        <a className="" href="#/people?sex=m">Male</a>
-        <a className="" href="#/people?sex=f">Female</a>
+        <SearchLink
+          params={{ sex: null }}
+          className={classNames({ 'is-active': !sex })}
+        >
+          All
+        </SearchLink>
+
+        <SearchLink
+          params={{ sex: 'm' }}
+          className={classNames({ 'is-active': sex === 'm' })}
+        >
+          Male
+        </SearchLink>
+
+        <SearchLink
+          params={{ sex: 'f' }}
+          className={classNames({ 'is-active': sex === 'f' })}
+        >
+          Female
+        </SearchLink>
```

Comment:
Might be, but if you have 3+ items you can separate business' logic (data) from UI (JSX):
1. 
```
export const getSearchLinks = (sex: SexTypes) => [
  { name: 'All', sex: null, isActive: !sex },
  { name: 'Male', sex: SexTypes.Male, isActive: sex === SexTypes.Male },
  { name: 'Female', sex: SexTypes.Female, isActive: sex === SexTypes.Female},
]  

```
2. 
```
<>
  {
    getSearchLinks(sex).map(({ name, sex: linkSex, isActive }) => (
       <SearchLink
          key={name}
          params={{ sex: linkSex }}
          className={classNames({ 'is-active': sex === linkSex })}
        >
          {name}
        </SearchLink>
    ))
  }
</>
```